templates/taxi/driver_list.html

```diff
@@ -6,8 +6,13 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
+        {% if driver.username == user.username %}
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Ensure that `driver.id` corresponds to the primary key of the `Driver` model. If the `Driver` model uses a different field for the primary key, this might lead to incorrect URL generation.